umm-spec-lib/test/cmr/umm_spec/test/gran_spatial_rep_alt_xpath_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns cmr.umm-spec.test.gran-spatial-rep-alt-xpath-test
+  "Tests getting GranuleSpatialRepresentation from alternative xpath for ISO19115." 
+  (:require
+   [clojure.java.io :as io]
+   [clojure.test :refer :all]
+   [cmr.umm-spec.json-schema :as js]
```

Comment:
nitpick: make this alias `json-schema` rather than `js`, and `location-keyword-helper` rather than `lkt`

Reply:
Done.